job_controller/status.go

```diff
@@ -0,0 +1,146 @@
+package job_controller
+
+import (
+	common "github.com/kubeflow/common/operator/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// JobCreatedReason is added in a job when it is created.
+	JobCreatedReason = "JobCreated"
+	// JobSucceededReason is added in a job when it is succeeded.
+	JobSucceededReason = "JobSucceeded"
+	// JobRunningReason is added in a job when it is running.
+	JobRunningReason = "JobRunning"
+	// JobFailedReason is added in a job when it is failed.
+	JobFailedReason = "JobFailed"
+	// JobRestarting is added in a job when it is restarting.
+	JobRestartingReason = "JobRestarting"
+
+	// labels for pods and servers.
+	ReplicaTypeLabel  = "replica-type"
+	ReplicaIndexLabel = "replica-index"
+	labelGroupName      = "group-name"
```

Comment:
We define the label as private but it is not used in the package, should we define it as a global var or just delete it?

Reply:
This is used in the newly added TestJobController